### DISCUSSION TITLE
考虑支持meili新出的向量搜索功能

### DISCUSSION THREAD

**Discussion by ZimaBlueee** [lrvinye/meilisearch-ui#156] [Ideas]
作者的这个项目太好用了，一开始我还以为你已经弃坑了，发现这个月更新了这么多真的很惊喜！！！

虽然meili的向量搜索还在beta，但实测已经非常稳定了，在当前ai的大流行环境下也很受欢迎。

请问咱们的项目考虑支持一下meili向量化和向量搜索面板吗？

**Comment by eyeix** [comment]
Meilisearch 的向量搜索支持稳定版后考虑集成，目前这个管理面板专注于 FTS.

**Reply by Jengro777** [reply]
支持
